Patches/AnimalCollabProj/AnimalCollabProj_CE_Patch_Races_Animal_DuckBilledPlatypus.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+
+			<li Class="CombatExtended.PatchOperationFindMod">
+				<modName>AnimalCollabProj</modName>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="ACPDuckBilledPlatypus"]/statBases</xpath>
+				<value>
+					<MeleeDodgeChance>0.07</MeleeDodgeChance>
+					<MeleeCritChance>0.11</MeleeCritChance>
+					<MeleeParryChance>0.02</MeleeParryChance>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="ACPDuckBilledPlatypus"]/tools</xpath>
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+							<label>spurs</label>
+							<capacities>
+								<li>PlatypusSting</li>
+							</capacities>
+							<power>7</power>
+							<cooldownTime>1.9</cooldownTime>
+							<linkedBodyPartsGroup>Spurs</linkedBodyPartsGroup>
+							<armorPenetrationSharp>0.4</armorPenetrationSharp>
```

Comment:
Might check the sharp AP here, since a lion bite only pierces 0.29mm.

Reply:
Whoops!

Fixed in 8fa8468